.github/workflows/linux-qe.yml

```diff
@@ -0,0 +1,150 @@
+name: tester-linux
+
+on:
+  workflow_run:
+    workflows: [Build Windows artifacts]
+    types:
+      - completed
+
+jobs:
+  tests:
+    runs-on: [self-hosted, linux, testing-farm]
```

Comment:
Similar to ci-definitions for crc-builder we want to ensure the previous flow has been completed and it is successful https://github.com/crc-org/ci-definitions/blob/7cb907719aef4898ef4eb0ec02d9124f5212d9d4/.github/workflows/crc-builder-pusher.yml#L12